api/operator/v1alpha1/certmanager_types.go

```diff
@@ -77,6 +77,10 @@ type DeploymentConfig struct {
 	// +kubebuilder:validation:Optional
 	// +optional
 	OverrideLabels map[string]string `json:"overrideLabels,omitempty"`
+
+	// +kubebuilder:validation:Optional
+	// +optional
+	OverrideResources corev1.ResourceRequirements `json:"overrideResources,omitempty"`
```

Comment:
Can we change this type to use a self-created struct that just contains Resources and Limits (essentially stripping out Claims as well, which would be counter-intuitive for discouraging users about adding any Claims as well).

It's better to not rely upon upstream type, rather have a copied one with only the necessary fields, xref: https://github.com/openshift/api/pull/1387#discussion_r1182817836